Update information on dokieli's data conventions and conformance to specifications (#380)
solid/solid-panes: Documentation/conventions.md

## Dokieli Document

> [dokieli](https://dokie.li/) is a clientside editor for decentralised article publishing, annotations and social interactions. Source code is available from https://github.com/linkeddata/dokieli .

Reviewer: ```suggestion
[dokieli](https://dokie.li/) is a clientside editor for decentralised article publishing, annotations, and social interactions. Source code is available from <https://github.com/linkeddata/dokieli>.
```

> Dokieli allows content creators to generate human- and machine-readable documents in HTML+RDFa, optionally embedding "data blocks" using the `<script type="">` mechanism to include content in Turtle, JSON-LD, and TriG. Dokieli also employs mechanisms to use media types that are acceptable by a server, e.g., JSON-LD, Turtle, HTML+RDFa, when for example generating data for servers conforming to the [Solid Protocol](https://solidproject.org/ED/protocol), [Linked Data Platform](https://www.w3.org/TR/ldp/), [Linked Data Notifications](https://www.w3.org/TR/ldn/), [Web Annotation Protocol](https://www.w3.org/TR/annotation-protocol), or [ActivityPub](https://www.w3.org/TR/activitypub/).

Reviewer: Will dokieli magically support embedded YAML-LD data blocks upon finalization of that spec? Or will you need to adjust your code? (Please consider this a "heads-up" that YAML-LD is coming, if you weren't already aware.)
```suggestion
dokieli allows content creators to generate human- and machine-readable documents in HTML+RDFa, optionally embedding "data blocks" using the `<script type="">` mechanism to include content in Turtle, JSON-LD, and TriG. dokieli also employs mechanisms to use media types that are acceptable by a server (e.g., JSON-LD, Turtle, HTML+RDFa) when, for example, generating data for servers conforming to the [Solid Protocol](https://solidproject.org/ED/protocol), [Linked Data Platform](https://www.w3.org/TR/ldp/), [Linked Data Notifications](https://www.w3.org/TR/ldn/), [Web Annotation Protocol](https://www.w3.org/TR/annotation-protocol), or [ActivityPub](https://www.w3.org/TR/activitypub/).
```

> The documents that are generated by dokieli are entirely *Linked Data* documents in that all significant units of information is available in triple statements (as per data models it conforms to), in addition to being accessible by text-based web browsers or screen readers, especially in the case of HTML.

Reviewer: ```suggestion
The documents that are generated by dokieli are entirely *Linked Data* documents, in that all significant units of information are available in triple statements (as per data models it conforms to), in addition to being accessible by text-based web browsers or screen readers, especially in the case of HTML.
```

> Dokieli can parse, use, or generate RDF sources including (but not limited to):

Reviewer: ```suggestion
dokieli can parse, use, or generate RDF sources including (but not limited to):
```

* [WebID](https://www.w3.org/2005/Incubator/webid/spec/identity/) for personal/agent identities.

> * [Web Access Control](https://solidproject.org/TR/wac)/ACL to set permissions on Web resources.

Reviewer: ```suggestion
* [Web Access Control (WAC)](https://solidproject.org/TR/wac)/[Access Control Lists (ACL)](https://dbpedia.org/resource/Access_Control_List) to set permissions on Web resources.
```

> * W3C [Web Annotation Model](https://www.w3.org/TR/annotation-model/), W3C [Web Annotation Vocabulary](https://www.w3.org/TR/annotation-vocab/), W3C [Embedding Web Annotations in HTML](https://www.w3.org/TR/annotation-html), and W3C [Selectors and States](https://www.w3.org/TR/selectors-states/) to model and identify annotations (eg. replies, peer-reviews, liking, resharing, bookmarking)

Reviewer: ```suggestion
* W3C [Web Annotation Model](https://www.w3.org/TR/annotation-model/), W3C [Web Annotation Vocabulary](https://www.w3.org/TR/annotation-vocab/), W3C [Embedding Web Annotations in HTML](https://www.w3.org/TR/annotation-html), and W3C [Selectors and States](https://www.w3.org/TR/selectors-states/) to model and identify annotations (e.g., replies, peer-reviews, liking, resharing, bookmarking)
```

* W3C [Activity Streams 2.0 vocabulary](https://www.w3.org/TR/activitystreams-vocabulary) for social activities.
* W3C [ODRL Information Model](https://www.w3.org/TR/odrl-model/) and W3C [ODRL Vocabulary & Expression](https://www.w3.org/TR/odrl-vocab/) to represent statements about the usage of content and services.

> * [Memento](https://tools.ietf.org/html/rfc7089) for resource management eg. TimeMap.

Reviewer: ```suggestion
* [Memento](https://tools.ietf.org/html/rfc7089) for resource management, e.g., TimeMap.
```

* [Creative Commons](https://creativecommons.org/) to assign license to individual contributions and annotations.

> * [Robust Links](http://robustlinks.mementoweb.org/) for hyperlinks eg. citations, and to show Link Decoration.

Reviewer: ```suggestion
* [Robust Links](http://robustlinks.mementoweb.org/) for hyperlinks (e.g., citations) and to show Link Decoration.
```

* [schema.org](http://schema.org/), [SPAR Ontologies](http://www.sparontologies.net/), [PROV-O](https://www.w3.org/TR/prov-o/), and various other vocabularies.

> Dokieli can also be used from any spawned instance, e.g., https://dokie.li/ , https://csarven.ca/, or as a Web browser extension. See also https://dokie.li/docs and https://github.com/linkeddata/dokieli/blob/main/README.md#specifications for further documentation on data conventions and list of specifications (protocols and data models) that dokieli conforms to.

Reviewer: ```suggestion
Dokieli can also be used from any spawned instance, e.g., https://dokie.li/ , https://csarven.ca/, or as a Web browser extension. See https://dokie.li/docs and https://github.com/linkeddata/dokieli/blob/main/README.md#specifications for further documentation on data conventions and lists of specifications (protocols and data models) that dokieli conforms to.
```
